test/langtools/tools/javac/records/RecordCompilationTests.java

```diff
@@ -145,8 +145,10 @@ public boolean process(Set<? extends TypeElement> annotations, RoundEnvironment
         }
     }
 
+    boolean useAP;
+
     public RecordCompilationTests() {
-        boolean useAP = System.getProperty("useAP") == null ? false : System.getProperty("useAP").equals("true");
+        useAP = System.getProperty("useAP") == null ? false : System.getProperty("useAP").equals("true");
```

Comment:
I know this is unrelated to your change, but this line could be improved with 

    useAP = System.getProperty("useAP", "false").equals("true")